src/guide/migration/listeners-removed.md

```diff
@@ -0,0 +1,77 @@
+---
+title: $listeners removed
+badges:
+  - breaking
+---
+
+# `$listeners` removed <MigrationBadges :badges="$frontmatter.badges" />
+
+## Overview
+
+the `$listeners` object has been removed in Vue 3. Event Listeners are now part of `$attrs`.
+
+```javascript
+{
+
+  text: 'this is a prop',
+  onClose: () => console.log('close Event triggered')
+}
+```
+
+## 2.x Syntax
+
+In Vue 2, you can access attributes passed to your components with `this.$attrs`, and event listeners with `this.$listeners`.
+
+These are usually Used in components that receive the attributes and listeners from its parents passed them on to and distinct element, usually in combination with `inheritAttrs: false`:
```

Comment:
How about this for the last sentence:

> In combination with `inheritAttrs: false`, they allow the developer to apply these attributes and listeners to some other element instead of the root element